Sources/Utilities/KMZendeskChatHandler.swift

```diff
@@ -15,41 +15,60 @@ struct KommunicateURL {
     static let dashboardURL = "https://dashboard.kommunicate.io/conversations/"
 }
 
-public class KMZendeskChatHandler {
-
-    public static let shared = KMZendeskChatHandler()
+public class KMZendeskChatHandler : NSObject, JWTAuthenticator {
+  
+    public func getToken(_ completion: @escaping (String?, Error?) -> Void) {
```

Comment:
Can we move it down? Also, it will be better to create a protocol and list all the public functions that we need to call from outside. For testing, we can replace with a dummy if that's the case and requirements will be clear.

Reply:
moved down. in next PR we will get clear idea of required public functions. Will create protocol there.